src/main/java/screens/GraphScene.java

```diff
@@ -81,16 +81,18 @@
      */
     public void drawGraph(final int id, final int radius) {
         if (radius < 5 || radius > 500) {
-            Stage newStage = this.FXElementsFactory.createStage();
-            Group group = this.FXElementsFactory.createGroup();
-            Label label = this.FXElementsFactory.createLabel("Radius is out of bounds");
+            Stage newStage = this.factory.createStage();
+            Group group = this.factory.createGroup();
+            Label label = this.factory.createLabel("Radius is out of bounds");
             group.getChildren().add(label);
-            Scene scene = this.FXElementsFactory.createScene(group, 150, 100);
-            this.FXElementsFactory.setScene(newStage, scene);
-            this.FXElementsFactory.show(newStage);
+            Scene scene = this.factory.createScene(group, 150, 100);
+            this.factory.setScene(newStage, scene);
+            this.factory.show(newStage);
             return;
```

Comment:
What is an empty return statement doing here? I see no loops to "break" out of which I'd say should be done with a break statement. Neither should a return statement be present in a void method.